### PR TITLE
Add all tags link

### PR DESCRIPTION
**Pull request by mnolan34**
Successfully able to cause a click of the allTags tag to open a proper search in the explore function

### CODE REVIEW

File: src/tests/tags-service.test.js [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    // Set up tests
+    beforeAll(async () => {
+        let promises = [];  // Initialize an empty list of promises
+
+        /*
+        Add promises to remove users with test username and
+        remove tuits with test texts to list of promises
+        */
+        promises.push(deleteUsersByUsername(sonic.username));
+        promises.push(deleteTuitByTuitText(mockedTuit1.tuit));
```
Comment (mcartervt) [inline src/tests/tags-service.test.js]: These are missing await, similar to last PR.  Just need to get current version from main.

File: src/tests/tags-service.test.js [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+        testTuit3 = await findTuitById(tid3);
+
+        let promises = [];  // Initialize an empty list of promises
+
+        /*
+        Add promises to remove users with test username and
+        remove tuits with test texts to list of promises
+        */
+        promises.push(deleteUsersByUsername(sonic.username));
+        promises.push(deleteTuitByTuitText(mockedTuit1.tuit));
```
Comment (mcartervt) [inline src/tests/tags-service.test.js]: These need await to match beforeAll.  See newest version in main.

**Review by mcartervt — CHANGES_REQUESTED** [review]
This PR is also missing tuit2tags-service.test.js in addition to tags-service.test.js changes.  I recommend repulling main into this branch.  

GUI tests worked as expected though!

**Comment by kachalmers** [comment]
Requesting to pull in changes that were merged into main via pull request #19 before approving.

**Comment by mnolan34** [comment]
Branch Main Merge made @mcartervt

**Review by kachalmers — APPROVED** [review]

**Review by mcartervt — APPROVED** [review]
Re-tested.  Can see my test files are in the branch correctly.  All Tags is showing the correct tuits associated with the tag. 

Approved!

**Pull request merged**